docs/node/run-your-node/advanced/sync-node-using-state-sync.md

```diff
@@ -151,43 +126,59 @@ The values you need are `latest_height` and `latest_hash` .
 Query our public Rosetta Gateway instance and obtain the trusted height and hash
 there:
 
-1. _TODO._
-
-#### Oasis Node's gRPC Endpoint
-
-Query our public Oasis node's gRPC endpoint and obtain the trusted height and
-hash there:
-
-1. _TODO._
-
-### Obtaining Addresses of Oasis Nodes' Publicly Exposed gRPC Endpoints
-
-To find the addresses of Oasis node's publicly exposed gRPC endpoints, use one
-of the following options.
-
-#### List Registered Nodes' Descriptors via Oasis CLI from the Local Oasis Node
-
-If you already have a local Oasis node set up, you can list the descriptors of
-all registered nodes via the [`oasis network show nodes`] Oasis CLI command.
+```bash
+curl -X POST https://rosetta.oasis.dev/api/block \
+-H "Content-Type: application/json" \
+-d '{
+    "network_identifier": {
+        "blockchain": "Oasis",
+        "network": "b11b369e0da5bb230b220127f5e7b242d385ef8c6f54906243f30af63c815535"
```

Comment:
Maybe mention that this is the chain context and where to look it up (links for mainnet and testnet). Also we should update the example after the Eden upgrade to include the new chain context.

Reply:
Added